src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -71,51 +71,67 @@ public Command parseCommand(String userInput) throws ParseException, CommandExce
         switch (commandWord) {
 
         case AddCommand.COMMAND_WORD:
+        case AddCommand.COMMAND_WORD_SHORT:
             return new AddCommandParser().parse(arguments);
 
         case DeleteCommand.COMMAND_WORD:
+        case DeleteCommand.COMMAND_WORD_SHORT:
             return new DeleteCommandParser().parse(arguments);
 
         case ClearCommand.COMMAND_WORD:
+        case ClearCommand.COMMAND_WORD_SHORT:
             return new ClearCommand();
 
         case FindCommand.COMMAND_WORD:
+        case FindCommand.COMMAND_WORD_SHORT:
             return new FindCommandParser().parse(arguments);
 
         case ListCommand.COMMAND_WORD:
+        case ListCommand.COMMAND_WORD_SHORT:
             return new ListCommand();
 
         case ExitCommand.COMMAND_WORD:
+        case ExitCommand.COMMAND_WORD_SHORT:
             return new ExitCommand();
 
         case HelpCommand.COMMAND_WORD:
+        case HelpCommand.COMMAND_WORD_SHORT:
             return new HelpCommand();
 
         case AddRelationshipCommand.COMMAND_WORD:
+        case AddRelationshipCommand.COMMAND_WORD_SHORT:
             return new AddRelationshipCommandParser().parse(arguments.trim());
 
         case DeleteRelationshipCommand.COMMAND_WORD:
```

Comment:
I liked how you use case fall through pretty smart

Reply:
thanks ik i am